### ISSUE TITLE
label autoencoder for outlier detection on the training set

### ISSUE THREAD

**Issue by bw4sz** [weecology/EvergladesSpeciesModel#17]
Long term, the tools we are building for the tree model need to be applied to the training data to detect mislabeled or anomalous birds. For the test dataset, it is small enough to just have Lindsey review it.
